client/src/components/SignUp/Types/SignUpTypes.js

```diff
@@ -62,13 +63,13 @@ class SignUpTypes extends React.Component {
 
   componentDidMount = () => {
 
-    // If a user is signed in and on this page, re-direct th
+    // If a user is signed in and on this page, re-direct them to settings
     const user = firebase.auth().currentUser;
     if (user !== null) {
       if (this.props.currentSignedInUser) {
         this.props.history.push('/settings')
       } else {

```

Comment:
If I'm not mistaken, you should only need an `if` statement here so you can actually remove the `else` part of it.

I'm sure it's there for debugging, but in the future polishing updates we can just remove it.